### PR TITLE
Feature/help text

### PR DESCRIPTION
**Pull request by bbonf**

### CODE REVIEW

File: frontend/src/app/routes.ts [hunk 1]
```diff
@@ -12,6 +15,14 @@ const routes: Routes = [
         path: 'spindle',
         component: SpindleComponent,
     },
+    {
```
Comment (XanderVertegaal) [inline frontend/src/app/routes.ts:18]: These are perhaps more specifically defined as child routes:
```
    {
        path: "spindle",
        children: [
            {
                path: "about",
                component: SpindleAboutComponent,
            },
            {
                path: "notation",
                component: SpindleNotationComponent,
            },
            {
                path: "",
                component: SpindleComponent,
            },
        ],
    },
```

File: frontend/src/app/spindle/spindle-notation/spindle-notation.component.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
+dt {
+    font-weight: bold;
+}
+
+.mono {
+    font-family: monospace;
+}
+
+var {
```
Comment (XanderVertegaal) [inline frontend/src/app/spindle/spindle-notation/spindle-notation.component.scss:9]: First time I see a use for the `<var>` element! Nice :)

File: frontend/src/app/spindle/spindle.component.html [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+        <p i18n>
+            Spindle is a neurosymbolic typelogical parser for Dutch.
+        </p>
+        <p i18n>Upon entering a Dutch phrase,
+            Spindle returns its analysis in the form of a lambda term that records the steps of its
+            derivation. Derivations are driven by the type formulas assigned to words. These formulas
+            determine how words combine into larger phrases.
+        </p>
+        <p i18n>
+            More info and references can be found under <a [routerLink]="['about']">About</a>. The notations used are explained
```
Comment (XanderVertegaal) [inline frontend/src/app/spindle/spindle.component.html:14]: References are not really found under `/about`, but under `/refs`.

File: frontend/src/app/spindle/spindle-about/spindle-about.component.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<div class="content is-size-5">
```
Comment (XanderVertegaal) [inline frontend/src/app/spindle/spindle-about/spindle-about.component.html:1]: Do we need the `is-size-5`? It makes the text on this page really stand out from the rest of the website, where the base text is smaller.

**Review by XanderVertegaal — CHANGES_REQUESTED** [review]
Well done! ⛏ My only substantial comment is the one about where to find the `references`. The other points are non-blocking suggestions or requests for clarification.

File: frontend/src/app/spindle/spindle-notation/spindle-notation.component.scss (hunk 2 above)
Comment (bbonf) [reply frontend/src/app/spindle/spindle-notation/spindle-notation.component.scss:9]: I also learned about it just today

**Pull request merged**